modules/hcp-aws-deploy-hc.adoc

```diff
@@ -43,6 +44,26 @@ $ hcp create cluster aws \
 <7> Specify the node pool replica count, for example, `3`.
 <8> By default, all `HostedCluster` and `NodePool` custom resources are created in the `clusters` namespace. You can use the `--namespace <namespace>` parameter, to create the `HostedCluster` and `NodePool` custom resources in a specific namespace.
 <9> Specify the Amazon Resource Name (ARN), for example, `arn:aws:iam::820196288204:role/myrole`.
+<10> If you want to indicate whether the EC2 instance runs on shared or single tenant hardware, include this field. The `--render-into` flag renders Kubernetes resources into the YAML file that you specify in this field. Then, continue to the next step to edit the YAML file.
+
+. In the command to create the hosted cluster, if you included the `--render-into` flag and specified a YAML file, edit the `NodePool` specification in the YAML file to indicate whether the EC2 instance should run on shared or single-tenant hardware.
```

Comment:
I find the following easier to understand:
```suggestion
.  If you included the `--render-into` flag in the `<previous_or_preceding>` command and specified a YAML file, edit the `NodePool` specification in the YAML file. You must indicate whether the EC2 instance should run on shared or single-tenant hardware, similar to the following example:
```
As written, I read the directive as "In the command, edit the spec in the resulting YAML file, if you used the flag". Which isn't correct, right? I think you mean to say, "If you used the flag in the command, edit the spec in the resulting YAML file."

Does that make sense? Feel free to ping me if I need to clarify.

Reply:
That makes sense. I like your suggested revision. Thanks!